tools/benchmark_event/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+add_definitions(-DG_LOG_DOMAIN="oio.tools.bench")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Werror")
+
+include_directories(BEFORE . ../..
+		${CMAKE_CURRENT_BINARY_DIR})
+
+add_executable(
+	bench_event_agent
```

Comment:
All our executable files use dashes as word separator. The `bean_prefix` macro may help you. The executable file should be something like `oio-event-benchmark`. The directory has to be renamed too.